tests/Behat/Content/Node/ProjectContentContext.php

```diff
@@ -97,4 +112,50 @@ public function alterProjectFields(BeforeParseEntityFieldsScope $scope): void {
     ]);
   }
 
+  /**
+   * Creates media Remote video with the specified URLs.
+   *
+   * Usage example:
+   *
+   * Given the following remote videos:
+   *   | url   |
+   *   | url 1 |
+   *   |  ...  |
+   *
+   * @Given the following remote video(s):
+   */
+  public function createMediaRemoteVideo(TableNode $table): void {
+    foreach ($table->getColumnsHash() as $hash) {
+      $media = \Drupal::entityTypeManager()
+        ->getStorage('media')->create([
+          'bundle' => 'remote_video',
+          'oe_media_oembed_video' => $hash['url'],
+          'status' => 1,
+        ]);
+      $media->save();
+
+      // Store for cleanup.
+      $this->media[] = $media;
+    }
+  }
+
+  /**
+   * Remove any created media.
+   *
+   * @AfterScenario
+   */
+  public function cleanMedias(): void {
```

Comment:
Why did you take this function from MediaContext.php and insert it here?

Reply:
I wanted to move it as @yenyasinn indicated but didn't have permission